sdk/keyvault/keyvault-secrets/src/core/challengeBasedAuthenticationPolicy.ts

```diff
@@ -71,7 +78,7 @@ export class ChallengeBasedAuthenticationPolicy extends BaseRequestPolicy {
       let kv = item.split("=");
       if (kv[0].trim() == "resource") {
         // Remove the quotations around the string
-        let resource = kv[1].trim().replace(/['"]+/g, '');
+        let resource = kv[1].trim().replace(/['"]+/g, "");
```

Comment:
Might want to check that split returned a second element in case the URL is malformed, or is throwing the intended fallback in this case?

Reply:
I changed this file by mistake.